delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/utils/EventLogUtils.scala

```diff
@@ -27,4 +29,41 @@ object EventLogUtils {
           )
         ) >> UIO.pure(None)
     }
+
+  /**
+    * Provide extension methods for an EventLog of Envelope[Event]
+    *
+    * @param eventLog the eventLog
+    */
+  implicit class EventLogOps[E <: Event](eventLog: EventLog[Envelope[E]]) {
```

Comment:
This is a syntax on an `EventLog`. It should be on the syntax object in the same manner than is done elsewhere.

Here you should define the method:
```scala
def fetchStateAt[State, E <: Event](eventLog: EventLog[Envelope[E]], persistenceId: String, rev: Long, initialState: State, next: (State, E) => State)(implicit
        revLens: Lens[State, Long]
    ): IO[Long, State] = ???

```

And in the syntax you just call it.